llvm/docs/MIRLangRef.rst

```diff
@@ -540,6 +540,9 @@ Register Flags
 The table below shows all of the possible register flags along with the
 corresponding internal ``llvm::RegState`` representation:
 
+..
+   Keep this in sync with MachineInstrBuilder.h
```

Comment:
Does this appear in the generated output?

Reply:
No: https://github.com/llvm/llvm-project/blob/6dab9ff72a1410989553285490f2a077c707c855/llvm/docs/MIRLangRef.rst